aspnet/migration/http-modules/sample/Asp.Net4/Asp.Net4/Modules/AddLinesModule.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace Asp.Net4.Modules
+{
+    public class AddLinesModule : IHttpModule
```

Comment:
You removed the middleware version of this, did you still need the module version?